pkg/gardenlet/controller/shoot/shoot_care_control.go

```diff
@@ -222,7 +238,10 @@ func (c *defaultCareControl) Care(shootObj *gardencorev1beta1.Shoot, key string)
 		// Fetch seed conditions of shoot is a seed
 		func(ctx context.Context) error {
 			seedConditions, err = retrieveSeedConditions(ctx, botanist)
-			return err
+			if err != nil {
+				botanist.Logger.Errorf("Error retrieving seed conditions: %+v", err)
+			}
+			return nil
```

Comment:
The error returned was also not handled before, IISIC.
I thought about improving this, but I was wondering if the care controller should really error in case it can't retrieve the seed conditions. In that case, the health check results (which might have been executed successfully) would not be transported to the Shoot conditions. 
My thought now was, it makes sense to rather "loose" the seed conditions (and just log an error) than to not update the shoot health conditions with the latest results.
What's your opinion on that?

Reply:
Hm, yes, makes sense. Thank you!